### DISCUSSION TITLE
cant set bed above 100, rated for 110 - s1  pro ?

### DISCUSSION THREAD

**Discussion by hayden-t** [mriscoc/Ender3V2S1#261] [Q&A]
found the code for hot end 300 deg limit, but noticed my hotbed limited to 100 too ??

**Comment by mriscoc** [comment]
Officially, Creality absolute maximum for bed temperature is:
```cpp
#define BED_MAXTEMP      110
```
But also there is a overshoot protection:
```cpp
#define BED_OVERSHOOT    10   // (°C) Forbid temperatures over MAXTEMP - OVERSHOOT
```
So, the maximum usable bed temperature is (110-10) = 100 °C

Source: https://github.com/CrealityOfficial/Ender-3S1/blob/main/Marlin/Configuration.h

**Reply by hayden-t** [reply]
so this is something to correct in the firmware ? no way around it, as default firmware allows 110

**Reply by mriscoc** [reply]
Without an official response from Creality and their official sources is a little dangerous to increment the temperature limit. I advice you to request to them to release the real sources and configuration files for the S1 Pro. That said, you can compile a version with that limit incremented to 110 by changing `BED_MAXTEMP` to 120.

**Reply by hayden-t** [reply]
but isnt the fact that the official firmware allows setting bed to 110 deg enough proof that creality supports this ?

**Reply by hayden-t** [reply]
also ive never seen overshoot of a bed by much as its so big and slow compared with hotend, but thats another topic

**Reply by mriscoc** [reply]
> but isn't the fact that the official firmware allows setting bed to 110 deg enough proof that creality supports this ?

Where are the sources of that firmware?

**Reply by hayden-t** [reply]
why do you need sources when you can see the manufacturer desired outcome and specification ?

**Comment by mriscoc** [comment]
For test in your printer change your configuration.h file increasing the bed temperature to:
```cpp
#define BED_MAXTEMP      120
```
So the maximum selectable temperature will be: (120 max -10 overshoot) = 110 °C

**Reply by hayden-t** [reply]
i would have to compile ? havent done that yet, but should be able to work it out, ill check the wiki

**Reply by mriscoc** [reply]
For S1 Pro, start with the configuration files for the T13 version: https://github.com/mriscoc/Special_Configurations/releases/tag/AM3950T13

There is also the possibility to have sponsored versions: https://github.com/mriscoc/Special_Configurations/blob/main/README.md
